explorer/types.go

```diff
@@ -299,9 +299,11 @@ type HostScan struct {
 
 // Host represents a host and the information gathered from scanning it.
 type Host struct {
-	PublicKey   types.PublicKey `json:"publicKey"`
-	NetAddress  string          `json:"netAddress"`
-	CountryCode string          `json:"countryCode"`
+	PublicKey      types.PublicKey    `json:"publicKey"`
+	NetAddress     string             `json:"netAddress"`
+	V2NetAddresses []chain.NetAddress `json:"netAddresses"`
```

Comment:
Fields are a little awkward here. Is there no way to unify them?

Reply:
I don't think there is a good way to do it since v2 addresses include a protocol and because we serve rhp2 over one and rhp4 over the other, but I would make `netAddress` `omit empty` to make sure we drop that field from the response altogether for v2 hosts.